pyreleaseplugin/release.py

```diff
@@ -245,6 +254,18 @@ def finalize_options(self):
             self.description = clean_description(self.description) or get_description()
 
         self.push_to_remote = bool(self.push_to_remote)
+        self.repository = self.repository
+
+        print ("Options selected for ReleaseCommand: ")
```

Comment:
I'd nuke line 259 -- not necessary. And remove the space between `print` and the open paren.